spec/build_spec_granite.cr

```diff
@@ -2,66 +2,44 @@ require "./build_spec_helper"
 
 module Amber::CLI
   describe "building a generated app using granite model and slang template" do
+    cleanup
```

Comment:
these should go in a `before_suite` block.  https://crystal-lang.org/api/0.32.1/Spec.html#before_each(&block)-class-method

Reply:
I cleaned it up a little, I didn't use the Spec.before_each or before_suite because I didn't look necessary to get the tests running. Also the Spec.before_each will run for all specs so I was affriad it might break spec in other directories